src/ng-select/lib/ng-select.component.spec.ts

```diff
@@ -2348,20 +2348,20 @@ describe('NgSelectComponent', () => {
 			expect(fixture.debugElement.query(By.css('.ng-placeholder'))).toBeTruthy();
 		}));
 
-		it('should not display ng-placeholder if an item is selected', fakeAsync(  () => {
+		it('should not display ng-placeholder if an item is selected', fakeAsync(() => {
```

Comment:
The test name is ''should display ng-placeholder if an item is selected'  If the default behaviour of the fixed placeholder is to be visible